synapse/tests/test_lib_scrape.py

```diff
@@ -751,16 +752,6 @@ def test_scrape_basic(self):
         nodes.remove(('crypto:currency:address',
                       ('ada', 'addr1yx2fxv2umyhttkxyxp8x0dlpdt3k6cwng5pxj3jhsydzerkr0vd4msrxnuwnccdxlhdjar77j6lg0wypcc9uar5d2shs2z78ve')))
 
-    async def test_scrape_async(self):
```

Comment:
Is this a merge artifact?

Reply:
no, more like a diff artifact. I moved some things around and diff detected it this way.